server/devices/AirConditioner.cc

```diff
@@ -19,9 +19,10 @@ namespace smart_home {
         return mode;
     }
 
-    void AirConditioner::displayStatuses() const {
+    std::string AirConditioner::displayStatuses() const {
```

Comment:
your functions does 2 things, displays and "gets" the status. if I read its name, "displaySomething()", will I expect it to also return me something?